### PR TITLE
Mpc try

### PR DESCRIPTION
**Pull request by kkolli**
Looks good, but if we have time, i'm wondering if we should refactor? Multiplayer Controller is super bloated, might be a secondary thing for now.

### CODE REVIEW

**Comment by kkolli** [comment]
And why are we doing ENUM's in separate files?

**Comment by edchiou** [comment]
Because Swift/XCode is sometimes dumb and won't compile unless I move them elsewhere.

**Comment by choucalate** [comment]
hey sorry can someone code review the multiplayergameview and setupview controller and merge this in?  I need it to do the timer/counter stuff 

Thanks!

File: client/GameViewController.swift [hunk 1]
```diff
@@ -183,7 +184,7 @@ class GameViewController : UIViewController, SKPhysicsContactDelegate {
                     
                     numberNode.setNeighbor(opNode)
                     opNode.setNeighbor(numberNode)
-                    
+                    println("psych")
```
Comment (edchiou) [inline client/GameViewController.swift]: lol, somehow my random print statements made it to this branch. Delete this.

File: client/GameViewController.swift [hunk 2]
```diff
@@ -191,6 +192,7 @@ class GameViewController : UIViewController, SKPhysicsContactDelegate {
                     scene!.joinedNodeB = opNode
                 }else{
                     // if hitting all 3
+                    println("Kappa")
```
Comment (edchiou) [inline client/GameViewController.swift]: Delete this as well

**Comment by edchiou** [comment]
I'll merge the branch as soon as someone deletes those println statements.

**Comment by choucalate** [comment]
removed! :)

**Pull request merged**